src/frontend/components/Loader/index.tsx

```diff
@@ -4,17 +4,29 @@ import styled from 'styled-components';
 
 import { theme } from '../../utils/theme/theme';
 
+const PreloaderStyled = styled.div`
```

Comment:
`Styled` suffix didn't provide any information about the purpose of this component. `Preloader` should be enough in my opinion and `LoaderStyled` should be renamed in `Spinner`.